projects/manage-supervision-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/sentence/entity/AdditionalSentence.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.integrations.delius.sentence.entity
+
+import jakarta.persistence.*
+import org.hibernate.annotations.Immutable
+import org.hibernate.annotations.SQLRestriction
+import uk.gov.justice.digital.hmpps.integrations.delius.overview.entity.Event
+import uk.gov.justice.digital.hmpps.integrations.delius.referencedata.entity.ReferenceData
+
+@Immutable
+@Entity
+@Table(name = "additional_sentence")
+@SQLRestriction("soft_deleted = 0")
+class AdditionalSentence(
+
+    @Id
+    @Column(name = "additional_sentence_id")
+    val id: Long,
+
+    @Column(name = "length")
+    val length: Long? = null,
+
+    @Column(name = "amount")
+    val amount: Long? = null,
+
+    @Lob
+    @Column(name = "notes")
```

Comment:
I think should either use @Lob without @column (as notes field has the same name?) or when using the @column name use `@Column(name = "notes", columnDefinition = "clob")` instead of @Lob? Could cause problems when deployed?

Reply:
done